core.php

```diff
@@ -366,4 +321,187 @@
+    "prune_users"               => "Inactive user records can be automatically pruned. A user is considered inactive if they have not logged in or have any other activity for a given amount of time.",
+    "prune_tickets"             => "Inactive tickets can be automatically pruned. A ticket is considered inactive if it is resolved and has not had any activity for a given amount of time.",
+    "prune_organisations"       => "Inactive organisations can be automatically pruned. An organisation is considered inactive if it has no users linked with it and has not had any activity for a given amount of time.",
+    "prune_export"              => "User data exports are stored on the file system but may take up room, you can prune export files generated before a certain date.",
+    "automatically_prune"       => "Automatically delete",
+    "days_after_saved"          => "Records after",
+    "days_after_last_activity"  => "Inactive records after",
+    "record_permanent_delete"   => "The records will be permanently deleted. This will impact reports.",
+    "record_delete_relations"   => "The records will be permanently deleted along with the following related data: :relations. This will impact reports.",
+    "manually_prune"            => "Manually delete records created before",
+
+    /*
+     * 2.4.0
+     */
+    "enable_ssl_warning"        => "If you enable this setting when the page doesn't load correctly, you may be locked out of SupportPal!",
+    "verify_frontend_loads"     => "Please verify that the frontend loads correctly below.",
+    "twig_operator_signature"   => "{{ operator.signature }} sammanslagningsfält kommer att bearbetas under körning så förhandsvisningen kan vara inkorrekt.",
+
+    /*
+     * 2.4.1
+     */
+    "additional_brands_desc"    => "Allows running other brands on your SupportPal installation.",
+    "no_branding"               => "Ingen varumärkning",
+    "no_branding_desc"          => "Removes SupportPal branding from the client-facing end of the help desk.",
+    "multi_ip"                  => "Multi-IP Support",
+    "multi_ip_desc"             => "Allows locking SupportPal to more than one internal/external IP.",
+    "purchase"                  => "Purchase",
+    "ticket_number_missing"     => "For ticket related emails, the {{ ticket.number }} merge field should be present in the email subject and must be wrapped in either <em>[#{{ ticket.number }}]</em> or <em>(#{{ ticket.number }})</em> in order to route replies to the correct ticket. If you would like to remove the ticket number, please consider enabling <a href='https://docs.supportpal.com/current/Email+Channel#SubAddressConfiguration' target='_blank'>email sub-addressing</a>.",
+
+    /*
+     * 2.5.0
+     */
+    "debug_mode_desc"           => "Aktivera felsökningsläge för att visa fel. Använd enbart om du felsöker eller instruerad av support. Fel lagras annars i /storage/logs.",
```

Comment:
It seems to have used the old translation for this, should really have just copied the English one as it changed since this was written.

Reply:
I think this is difficult to handle - I'm not sure how to handle this case.

The script is incredibly naive, it just does the following:
1. Read the english file into string
2. Loop over the translated file array
3. Use regex to identify position in the english file
4. Copy the translated value into the english file

Reply:
Maybe can do some post processing. Diff the before and after contents, looking specifically for new additions (`+`), and ensure all additions in English.

Reply:
I don't think that will work. It would need to identify the version block and only add the translation if it's the same block.

Reply:
Okay will manually handle for now.